### PR TITLE
support k8s 1.30

### PR DESCRIPTION
**Pull request by YZ775**

### CODE REVIEW

File: Makefile.versions [hunk 1]
```diff
@@ -1,4 +1,4 @@
-E2ETEST_K8S_VERSION  := 1.29.5
+ENVTEST_K8S_VERSION=1.31.0
```
Comment (chez-shanpu) [inline Makefile.versions]:
```suggestion
ENVTEST_K8S_VERSION=1.30.3
```
Is it too new?

**Review by chez-shanpu — COMMENTED** [review]
Almost good :D
I left a small comment.

File: Makefile.versions (hunk 1 above)
Comment (YZ775) [reply Makefile.versions]: Thank you! I fixed it.

**Review by chez-shanpu — APPROVED** [review]
LGTM!

**Pull request merged**